velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -14,8 +14,10 @@
  * limitations under the License.
  */
 
+#include <boost/algorithm/string.hpp>
```

Comment:
Are these includes needed?

Reply:
CI Compilation would fail without this include:

```
2024-03-06T15:38:41.3214677Z       In file included from /project/velox/functions/sparksql/DateTimeFunctions.cpp:17:
2024-03-06T15:38:41.3215875Z       /project/./velox/functions/sparksql/DateTimeFunctions.h: In static member function ‘static std::optional<signed char> facebook::velox::functions::sparksql::NextDayFunction<T>::getDayOfWeekFromString(const facebook::velox::StringView&)’:
2024-03-06T15:38:41.3216354Z       /project/./velox/functions/sparksql/DateTimeFunctions.h:583:9: error: ‘boost’ has not been declared
2024-03-06T15:38:41.3216558Z         583 |         boost::algorithm::to_lower_copy(dayOfWeek.str());
2024-03-06T15:38:41.3216661Z             |         ^~~~~
```